text/game/abilities.stringtable

```diff
@@ -1406,7 +1406,7 @@
     </Entry>
     <Entry>
       <ID>294</ID>
-      <DefaultText>Schwingt die Nahkampfwaffe des Kämpfers in einem großen Bogen und fügt Feinden den Niederschlag Effekt zu.</DefaultText>
+      <DefaultText>Schwingt die Nahkampfwaffe des Kämpfers in einem großen Bogen und fügt Feinden den Niederstrecken Effekt zu.</DefaultText>
```

Comment:
Jein... Glaub das war in vielen Fällen wegen der Verlinkung grossgeschrieben auch wenn es von der Grammatik her falsch ist. Wenn kein Tooltip erscheint, dann könnten wir es klein schreiben.